flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobMaster.java

```diff
@@ -963,6 +1120,36 @@ private Acknowledge suspendExecution(final Exception cause) {
 		return Acknowledge.get();
 	}
 
+	/**
+	 * Schedules the execution of the current {@link ExecutionGraph}.
+	 */
+	private void scheduleExecutionGraph() {
+		try {
+			executionGraph.scheduleForExecution();
+		}
+		catch (Throwable t) {
+			executionGraph.failGlobal(t);
+		}
+	}
+
+	/**
+	 * Dispose the savepoint stored under the given path.
+	 *
+	 * @param savepointPath path where the savepoint is stored
+	 */
+	private void disposeSavepoint(String savepointPath) {
+		try {
+			// delete the temporary savepoint
+			Checkpoints.disposeSavepoint(
+				savepointPath,
+				jobMasterConfiguration.getConfiguration(),
+				userCodeLoader,
+				log);
+		} catch (FlinkException | IOException de) {
```

Comment:
Why `de`? Just want to make sure it's not a typo.

Reply:
is a typo. Will change it.